executor/aggregate_test.go

```diff
@@ -752,3 +752,43 @@ func (s *testSuite1) TestIssue10608(c *C) {
 	tk.MustQuery("select (select group_concat(concat(123,'-')) from t where t.a = s.b group by t.a) as t from s;").Check(testkit.Rows("123-", "123-"))
 
 }
+
+func (s *testSuiteAgg) TestIssue12759HashAggCalledByApply(c *C) {
```

Comment:
This test case is useless in 3.0 branch, see https://github.com/pingcap/tidb/pull/12766#issuecomment-542678906